src/MetalPerformanceShaders/MPSKernel.cs

```diff
@@ -250,6 +271,54 @@ public MPSImageGaussianPyramid (IMTLDevice device, nuint kernelWidth, nuint kern
 			}
 		}
 	}
+
+	public partial class MPSImageLaplacianPyramid {
+		[DesignatedInitializer]
+		public MPSImageLaplacianPyramid (IMTLDevice device, nuint kernelWidth, nuint kernelHeight, float[] kernelWeights) : base (NSObjectFlag.Empty)
+		{
+			if (kernelWeights == null)
+				throw new ArgumentNullException (nameof (kernelWeights));
+			if (kernelWidth * kernelHeight < (nuint) kernelWeights.Length)
```

Comment:
I think you got the comparison wrong: it should be the other way here (`kernelWidth * kernelHeight > (nuint) kernelWeights.Length)`